### PR TITLE
Add scroll to top

### LINKED ISSUE #986: E-commerce scroll bound animations
1) sticky toolbar that appears after scrolling: https://screenshot.googleplex.com/sZk2JFfCOGX
2) Scroll to top button that appears upon scrolling

### PR DESCRIPTION
**Pull request by kul3r4**
Partially fixes #986

### CODE REVIEW

**Comment by kul3r4** [comment]
@sebastianbenz Actually please wait to review as I want to check if i can use amp-position-observer to show and hide the button

**Comment by kul3r4** [comment]
@sebastianbenz PTAL
@aghassemi thanks for your help!

**Comment by sebastianbenz** [comment]
I think this would actually be better in a separate sample. WDYT?

File: src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html [hunk 1]
```diff
@@ -98,7 +123,64 @@
...
+</amp-animation>
+ <!-- We use `amp-position-observer` to start the animation when the user starts to scroll. -->
+<div id="marker">
+  <amp-position-observer
+     on="enter:hideAnim.start; exit:showAnim.start"
+     layout="nodisplay">
+</amp-position-observer>
+ </div>
+ <!-- We use the `scrollTo` action to scroll the page when the button is tapped. -->
+     <button id="scrollToTopButton" on="tap:top-header.scrollTo(duration=200)" class="scrollToTop">↑</button>
```
Comment (aghassemi) [inline src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html]: `⌃` maybe?

File: src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html [hunk 2]
```diff
@@ -98,7 +123,64 @@
...
+                 }
+               ]
+     }
+   </script>
+ </amp-animation>
+ <!-- ... and the second one is for adding the button.-->
+ <amp-animation id="hideAnim" layout="nodisplay">
+  <script type="application/json">
+    {
+      "duration": "200ms",
```
Comment (aghassemi) [inline src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html]: spacing is off (my bad from the sample, codepen does weird formatting)

**Comment by kul3r4** [comment]
@sebastianbenz Sure I can move it.
Would it be better into "Visual effects" or into "Advanced"?

**Comment by sebastianbenz** [comment]
I'd say Visual effects

File: src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html (hunk 2 above)
Comment (kul3r4) [reply src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html]: done!

File: src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html (hunk 1 above)
Comment (kul3r4) [reply src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html]: done!

**Comment by kul3r4** [comment]
@sebastianbenz I moved the sample.

File: src/50_Visual_Effects/Scroll_to_top.html [hunk 3]
```diff
@@ -0,0 +1,130 @@
+<!--
+  ## Introduction
+
+  [amp-position-observer](https://www.ampproject.org/docs/reference/components/amp-position-observer)
+  combined with [amp-animation](https://www.ampproject.org/docs/reference/components/amp-animation) could be used to
```
Comment (sebastianbenz) [inline src/50_Visual_Effects/Scroll_to_top.html]: s/could/can as you prove later ;-)

**Comment by kul3r4** [comment]
Done!

**Comment by sebastianbenz** [comment]
Great!

**Pull request merged**